pom.xml

```diff
@@ -85,7 +85,7 @@
                     <source>11</source>
                     <detectJavaApiLink>false</detectJavaApiLink>
                     <!-- Turn off Java 8 strict Javadoc checking -->
-<!--                    <additionalparam>-Xdoclint:none</additionalparam>-->
+<!--                <additionalparam>-Xdoclint:none</additionalparam>-->
```

Comment:
See line 87

Reply:
You are right, thank you for your guidance. I will make the necessary changes.